exercises/hamming/.meta/src/reference/java/Hamming.java

```diff
@@ -1,36 +1,26 @@
+import java.util.function.IntPredicate;
+import java.util.stream.IntStream;
+
 class Hamming {
-    private final int hammingDistance;
+    private final String leftStrand, rightStrand;
```

Comment:
Why store the left strand and right strand instead of just storing the hamming distance? As the distance won't change it seems more efficient to just compute it once? 🙂

Reply:
Hello @FridaTveit , thanks for the review. The reason is that, if we imagine  a Real Life situation, the Hamming Class could offer other methods than `getHammingDistance()`. I'll add that almost every solution submitted by students include the storage of both strands. But I agree that if we focus only on completing the exercise, it is not needed. I'll let you decide!

Reply:
I think it's best to optimise the current solution instead of considering possible other scenarios. If I were writing this code in real life I would only store the hamming distance. If at some point I needed to add other methods to the class that needed to use the individual strands then I would change my implementation. But there's no point changing it before it's needed 🙂

Reply:
Good point, I'll update the file accordingly!

Reply:
@FridaTveit, thanks for the REALLY quick answers ;-)
Changes commited!